Makefile

```diff
@@ -6,11 +7,13 @@ HOMEPAGE:="https://github.com/afiniate/ouija"
 DEV_REPO:="git@github.com:afiniate/ouija.git"
 BUG_REPORTS:="https://github.com/afiniate/ouija/issues"
 
-DESC:="Ouija is a path resolution library for ocaml"
+DESC_FILE:=$(CURDIR)/description
 
 BUILD_DEPS:=vrt
-DEPS:=core async async_unix async_shell cohttp cohttp.async atdgen sexplib cryptokit
+DEPS:=core async async_unix async_shell cohttp atdgen sexplib cryptokit
 
+all:
+	echo "${DESC}"
```

Comment:
Is this `echo` intentional? Also, `DESC` is no longer declared (now only `DESC_FILE` is)